auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -262,6 +273,14 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
     return LaunchDarklyService.getFlag(LDFlags.EnableMandatoryAddress) || false
   }
 
+  // open Name Request
+  private goToNameRequest (): void {
```

Comment:
is opening in a new window a requirement ? If you open in new window the session won't get sync'ed as the session storage values won't be present in new window.

Reply:
hmm ok, i was curious about this. 
The ticket DOES call out opening in a new tab explicitly, but if this is going to create a sync issue then perhaps we keep the same tab?

Reply:
You may need to discuss with the UX team and convince this issue :)

Reply:
Good catch, Sumesh. That was why NR UI was recently changed to open in the same window.

Reply:
Updated! Will post comment in ticket for UX, thanks!